docs/get_started/raspbian_setup.md

```diff
@@ -1,7 +1,7 @@
 # Installing MXNet on Raspbian
-MXNet currently supports the Debian based Raspbian operating system for Raspberry Pi Devices, offering a python API. 
+MXNet currently supports a python API for the Debian based Raspbian operating system for Raspberry Pi Devices.
```

Comment:
MXNet supports a Python API for the Debian-based Raspbian operating system so you can run MXNet on Raspberry Pi devices.

ED: This is an abrupt intro. Installation should be a step in the tutorial, not a separate topic.